addons/base_partner_address/__openerp__.py

```diff
@@ -30,11 +30,7 @@
     ],
     'data': [
         'security/ir.model.access.csv',
-        'views/assets.xml',
```

Comment:
@kaerdsar Please add 'view/assets.xml' again. See my comment in base_partner_address/static/src/css/base.css for more information.